package qatechiepavanwebtesting.day15;

public class HandleExceptionsDemo {


    public static void main (String args [])
    {
   /*     //ArithmeticException
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a number...");
        int num = sc.nextInt();

        try
        {
         System.out.println(100/num);
        }
        catch (ArithmeticException a)
        {
            System.out.println("You have entered an invalid input...."); //this will be printed if an invalid value is entered but ret of the program will run
        }

        System.out.println("Program is completed...");
        System.out.println("Program is exited...");



    */
    }
}